people/feeback.md

```diff
@@ -0,0 +1,64 @@
+# Feedback
+
+> [Giving feedback turns out to be the unnatural atomic building block atop which the unnatural skill set of management gets built](https://a16z.com/2012/10/17/making-yourself-a-ceo/)
+
+Feedback is important to maintain a shared understanding of what is going well and what needs improvement.
+
+It promotes sharing of ideas on how to maintain and develop a shared understanding into impactful action.
+
+It is an integral part to any iteratively improving system.
+
+---
+
+## Where will feedback be used
+
+Feedack will be used in 1:1s when necessary, at the end of each quarter in regards to PDRs, and as evidence for promotion and remuneration applications.
+
+---
+
+## What do we collect feedback on?
+
+The how and the what.
+
+We collect feedback across 3 lines of your work, technical, inter, and intra team working.
+
+How you do what you do, how you help others to do what they do, and what support is needed to help make you more effective.
+
+"Intra-team" is the smallest group you work with 1 branch removed from yourself. e.g. The project you're currently working on.
+
+"Inter-team" is any other groups outside of your intra-team.
+
+### Technical
+
+- How do you use your technical know-how to work towards the team's vision and achieve it's goals?
+- How do you use your technical know-how to help others work towards the team's vision and achieve it's goals?
+- Where can we support you to develop and share your technical skills?
+
+### Intra-team working
+
+- How are you working within your team to help the team work towards the team's vision and achieve it's goals?
+- Where can your team support you to help you be more effective within it?
+
+### Inter-team working
+
+- How are you working and collaborating with teams and others outside of your team to help make yours and their team more effective?
+- Where can others support you to be more effective to them?
+- Where can others support you to be more effective within your team?
+
+---
+
+## What qualities should feedback have?
+
+- **Conversational:** It is an opinion, not facts, and can be the beginning of a conversation rather than the end.
+- **Timely:** Allowing for all involved to respond.
+- **Future leaning:** Given and taken with a view of how it is applicable to a better future.
+
+---
+
+## When do we go through the feedback?
+
+As timely as possible making the feedback as easy as possible to act and respond to, but with regards to the fact that receiving feedback too often will cause a numbness to it, probably reduce the quality of it, and miss the opportunity to see patterns forming.
```

Comment:
Can we naively say once a month?

Reply:
```suggestion
As timely as possible making the feedback as easy as possible to act and respond to, but with regards to the fact that receiving feedback too often will cause a numbness to it, probably reduce the quality of it, and miss the opportunity to see patterns forming.

This will be initially be done every month, or in the context of fortnightly 1:1s, every second occurrence.
```